fwprovider/tests/resource_acl_test.go

```diff
@@ -0,0 +1,152 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package tests
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+)
+
+func TestAccAcl_User(t *testing.T) {
+	resourceName := "proxmox_virtual_environment_acl.test"
+
+	accProviders := testAccMuxProviders(context.Background(), t)
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: accProviders,
+		CheckDestroy:             nil,
+		Steps: []resource.TestStep{
```

Comment:
`provider_test@pve` need to be manually pre-created for the tests.

It's fine for now, but it would be better to make them self-sustained at some point. I.e. have a (random) test user created via API call at the beginning, perhaps in `PreConfig`, like
```go
		PreConfig: func() {
			ac := &access.Client{Client: getAPIClient()}
			ac.CreateUser(context.Background(), &access.UserCreateRequestBody{
				ID:       userID,
				Password: gofakeit.Password(true, true, true, true, false, 10),
			})
		},
```

Then the test user(s) need to be deleted at the end, preferably via `t.Cleanup()`